companion/lib/Data/StoreBase.ts

```diff
@@ -39,334 +48,244 @@ export class DataStoreBase {
 	/**
 	 * The config directory
 	 */
-	private readonly cfgDir: string = ''
+	public readonly cfgDir: string
 	/**
 	 * The full main file path
 	 */
 	private readonly cfgFile: string = ''
 	/**
-	 * The full temporary file path
+	 * The full main legacy file path
 	 */
-	private readonly cfgTmpFile: string = ''
+	private readonly cfgLegacyFile: string = ''
 	/**
-	 * The stored defaults for a new store
+	 * The default table to dumb keys when one isn't specified
```

Comment:
```suggestion
	 * The default table to dump keys when one isn't specified
```

Reply:
I mean, it could be poetic in the right context :)

Reply:
done